blaze-server/src/main/scala/org/http4s/blaze/server/BlazeServerBuilder.scala

```diff
@@ -209,6 +209,18 @@ class BlazeServerBuilder[F[_]] private (
   override def bindSocketAddress(socketAddress: InetSocketAddress): Self =
     copy(socketAddress = socketAddress)
 
+
+  /** Configures the compute thread pool used to process requests
```

Comment:
I’m not too happy with the wording here — “process requests” is pretty ambiguous. That said, I can’t think of a better term.

Reply:
I think it's good.  I came up with "responding to requests", but is that calling the database, or is that just serializing a response body?  My term is worse.